### PR TITLE
Added gulp+browserSync for #11

### PR DESCRIPTION
**Pull request by kengz**
Hello @georgschlenkhoff I was having lunch and thought why not do it now?

1) I added `browserSync` and `Gulp` for the worry-free browser live-preview, i.e. without having to install anything else. Only supplanted the original `watch` method at line 48 of `dokker.js` with the gulp task, which is written all in the gulpfile.js.
Tested it and it ran like a charm. Just do the `./node_modules/dokker/bin/watch` as usual. You can save your `.js` file and it will rebuild and reload automatically.
You might wish to update your README and drop the need to install Live Preview. Maybe the new description would be "just do `./node_modules/dokker/bin/watch` and magic will happen.

2) Fixed a build-breaking bug at line 390 which the `mocha.command`. It's not needed and the build system doesn't know what to do with it.

So basically we are at the next version of Dokker, with an awesome live-preview!

### CODE REVIEW

**Comment by kengz** [comment]
Feature request #11 for easier live preview is solved with this update.

**Comment by kengz** [comment]
3) I added a quick meta tag in HTML's head that attributes credits back to dokkerjs.

**Comment by kengz** [comment]
4) simplified and update the README: live preview, Dokker in the wild.

Btw you may wish to regenerate the docs for `dokkerjs.com` before publishing/updating the site. I haven't touch it in case I screw up any manual config you have.

**Comment by ghost** [comment]
Alright. Will merge. Will also grant to do some advertising for `lomath` for huge commit of you ;-)

**Comment by kengz** [comment]
;) thanks.
and I see you're using the new theme too!
Btw I'll keep testing and fixing any issues I found since I'll be using Dokker for my next projects.
